tpm2_pytss/FAPI.py

```diff
@@ -957,7 +957,9 @@ def quote(
             ffi.string(get_dptr(quote_info, lib.Fapi_Free)).decode(),
             bytes(ffi.unpack(get_dptr(signature, lib.Fapi_Free), signature_len[0])),
             ffi.string(get_dptr(pcr_log, lib.Fapi_Free)).decode(),
```

Comment:
Is it worth checking for NULL pointer here too? tss2-fapi returns a '\n' for now but it could change in the future and crash tpm2-pytss again.

Reply:
For the future I guess we need to check all FAPI calls that might return NULL

Reply:
Maybe make get_dptr smarter?